Objects/tupleobject.c

```diff
@@ -57,6 +57,10 @@ show_track(void)
     fprintf(stderr, "%.2f%% tuple tracking rate\n\n",
         (100.0*count_tracked/(count_untracked+count_tracked)));
 }
+
+#   define TUPLE_GC_TRACK(op) (count_tracked++, _PyObject_GC_TRACK(op))
```

Comment:
Can you please try to convert this macro to a static inline function instead? Something like:

```
static inline void TUPLE_GC_TRACK(PyTupleObject *op)
{
    count_tracked++;
    _PyObject_GC_TRACK(op);
}
```

I don't know if the parameter type should be `PyTupleObject*` or `PyObject*`.